schemas/paths/event-search.yaml

```diff
@@ -95,6 +95,106 @@ get:
+    - name: jailbroken
+      in: query
+      schema:
+        type: boolean
+      description: |
+        Filter events by jailbroken device detection.  
+        > Note: When using this parameter, only events with the `jailbroken` property explicitly set to `true` or `false` are returned. Events with undefined `jailbroken` property are left out of the response.
+    - name: frida
+      in: query
+      schema:
+        type: boolean
+      description: |
+        Filter events by Frida instrumentation detection.  
+        > Note: When using this parameter, only events with the `frida` property explicitly set to `true` or `false` are returned. Events with undefined `frida` property are left out of the response.
+    - name: factory_reset
+      in: query
+      schema:
+        type: boolean
+      description: |
+        Filter events by factory reset detection.  
+        > Note: When using this parameter, only events with the `factory_reset` property explicitly set to `true` or `false` are returned. Events with undefined `factory_reset` property are left out of the response.
+    - name: cloned_app
+      in: query
+      schema:
+        type: boolean
+      description: |
+        Filter events by cloned app detection.  
+        > Note: When using this parameter, only events with the `cloned_app` property explicitly set to `true` or `false` are returned. Events with undefined `cloned_app` property are left out of the response.
+    - name: emulator
+      in: query
+      schema:
+        type: boolean
+      description: |
+        Filter events by emulator usage.  
+        > Note: When using this parameter, only events with the `emulator` property explicitly set to `true` or `false` are returned. Events with undefined `emulator` property are left out of the response.
+    - name: root_apps
+      in: query
+      schema:
+        type: boolean
+      description: |
+        Filter events by root apps detection.  
+        > Note: When using this parameter, only events with the `root_apps` property explicitly set to `true` or `false` are returned. Events with undefined `root_apps` property are left out of the response.
+    - name: vpn_confidence
+      in: query
+      schema:
+        type: string
+        enum: [HIGH, MEDIUM, LOW]
+      description: |
+        Filter events by VPN confidence level.  
+        > Note: When using this parameter, only events with the `vpn_confidence` property explicitly set to a valid value are returned. Events with undefined `vpn_confidence` property are left out of the response.
+    - name: min_suspect_score
+      in: query
+      schema:
+        type: number
+        format: float
+      description: |
+        Minimum suspect score threshold for filtering events.  
+        > Note: When using this parameter, only events with the `min_suspect_score` property explicitly set to a valid value are returned. Events with undefined `min_suspect_score` property are left out of the response.
```

Comment:
There is no `min_suspect_score` property on the event object, not even deeply nested.